wrapanapi/systems/scvmm.py

```diff
@@ -746,11 +757,9 @@ def download_file(self, url, name, dest="L:\\Library\\VHDs\\", unzip=False):
             $wc = New-Object System.Net.WebClient
             $wc.DownloadFile($url, $output)
         """.format(url=url, name=name, dest=dest)
-        if unzip:
-            script += "Expand-Archive -LiteralPath $output -DestinationPath {dest}".format(
-                dest=dest
-            )
         self.run_script(script)
+        if unzip:
+            self.unzip_archive(dest + name, dest)
```

Comment:
Please use string formatting here, as both are parameters and may not be of compatible types for using `+`.

Reply:
You're welcome to use f-strings as #412 was merged.

Reply:
Agreed, thanks!

Reply:
Done